index.d.ts

```diff
@@ -0,0 +1,24 @@
+import * as http from "http"
+
+declare namespace vhost {
+  interface RequestHandler {
+    (req: http.IncomingMessage, res: http.ServerResponse, next: Function): void
+  }
+}
+
+declare function vhost(host: string, handler: vhost.RequestHandler): vhost.RequestHandler
```

Comment:
Ok, this was my last comment from yesterday: the `host` parameter also accepts a regular expression here. And should these parameters be named the same as they are in the docs and underlying JS? I assume users will see these names in TS errors?